### PR TITLE
Text change: 9002 Error Recovery Notice (EXPOSUREAPP-3106)

### PR DESCRIPTION
**Pull request by ralfgehrer**

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -1182,9 +1182,9 @@
...
     <string name="errors_generic_text_unknown_error_cause">"Ein unbekannter Fehler ist aufgetreten."</string>
     <!-- XTXT: error dialog - text when a catastrophic error occured from which the app recovered automatically via data reset -->
-    <string name="errors_generic_text_catastrophic_error_recovery_via_reset">"Aufgrund eines technischen Problems wurde Ihre App zurückgesetzt. Das hat jedoch keine Auswirkungen auf die Verwendung der App. Sie werden weiterhin über Risiko-Begegnungen benachrichtigt und können andere warnen, falls Sie positiv auf COVID-19 getestet wurden."</string>
+    <string name="errors_generic_text_catastrophic_error_recovery_via_reset">"Aufgrund eines technischen Problems wurde Ihre App zurückgesetzt. Dies hat keine Auswirkung auf die Risiko-Ermittlung und damit auf Ihren Risikostatus. Sie werden weiterhin über Risiko-Begegnungen benachrichtigt und können andere warnen, falls Sie positiv auf COVID-19 getestet wurden. Durch das Zurücksetzen der App gehen aber zuvor eingelesene QR-Codes für die elektronische Benachrichtigung über Ihre Testergebnisse verloren. Bitte wenden Sie sich an Ihren Hausarzt oder Labor, um Ihr Testergebnis zu erfahren."</string>
```
Comment (ralfgehrer) [inline Corona-Warn-App/src/main/res/values-de/strings.xml:1185]: @SabineLoss I dropped the last part of the text block "Detaillierte Informationen hierzu finden Sie in der [Link: FAQ]" as this text is shown within an alert dialog with a "Details" button that opens the link. For further info, please refer to the ticket or ping me.

**Comment by vaubaehn** [comment]
Thank you.
Related to https://github.com/corona-warn-app/cwa-app-android/pull/1254#issuecomment-700923352

**Review by harambasicluka — DISMISSED** [review]

**Review by harambasicluka — APPROVED** [review]

**Review by janetback — APPROVED** [review]
UA approved.

File: Corona-Warn-App/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (SabineLoss) [reply Corona-Warn-App/src/main/res/values-de/strings.xml:1185]: @ralfgehrer : Sure, that's fine with me.

**Review by SabineLoss — APPROVED** [review]

**Pull request merged**